tests/shared/FlowManagerSettingsGetTestDefs.js

```diff
@@ -0,0 +1,465 @@
+                                "cursor-size": 90,
+                                "text-scaling-factor": 0.75
+                            },
+                            "http://registry.gpii.net/applications/org.alsa-project": {
+                                "masterVolume": 50
+                            }
+                        }
+                    }
+                }
+            },
+            expectedMatchMakerOutput: {
+                "inferredConfiguration": {
+                    "gpii-default": {
+                        "applications": {
+                            "org.gnome.desktop.a11y.magnifier": {
+                                "active": true,
+                                "settings": {
+                                    "http://registry.gpii.net/common/magnification": 1.5,
+                                    "http://registry.gpii.net/common/magnifierPosition": "FullScreen",
+                                    "http://registry.gpii.net/applications/org.gnome.desktop.a11y.magnifier": {
+                                        "mag-factor": 1.5,
+                                        "screen-position": "full-screen"
+                                    }
+                                }
+                            },
+                            "org.gnome.desktop.interface": {
+                                "active": true,
+                                "settings": {
+                                    "http://registry.gpii.net/common/fontSize": 9,
+                                    "http://registry.gpii.net/common/cursorSize": 0.9,
+                                    "http://registry.gpii.net/applications/org.gnome.desktop.interface": {
+                                        "cursor-size": 90,
+                                        "text-scaling-factor": 0.75
+                                    }
+                                }
+                            },
+                            "org.alsa-project": {
+                                "active": true,
+                                "settings": {
+                                    "http://registry.gpii.net/common/volume": 0.5,
+                                    "http://registry.gpii.net/applications/org.alsa-project": {
+                                        "masterVolume": 50
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        disruptions: [{
+            sequenceGrade: "gpii.tests.cloud.oauth2.settingsGet.disruption.mainSequence"
+        }]
+    },
+
+    // Successful use cases that request settings for a non-existent GPII key with proper access tokens granted via Resource Owner GPII key grant
+    {
+        testDef: {
+            name: "A successful workflow for retrieving settings for an existing GPII key",
```

Comment:
This is my bad introduced into the master. Can you help to take the chance to replace the word "existing" in this message to "nonexistent"? Thanks.

Reply:
No problem.